deploy/rpm/SPECS/karavi-authorization.spec

```diff
@@ -18,7 +18,7 @@ cp deploy $RPM_BUILD_ROOT%{_tmppath}
 %post
 echo "Installing %{name}-%{version}.%{release}"
 %{_tmppath}/deploy
-echo "Installation Complete"
+echo "RPM Installation Complete"
```

Comment:
Can we mirror the "Installing... " text by indicating the name, version, release as well

Reply:
Done. Added name, version, and release